composer.json

```diff
@@ -39,7 +39,7 @@
     "evenement/evenement": "^2.0 | ^3.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5.7",
+    "phpunit/phpunit": "^8",
```

Comment:
Changes LGTM, but this implies this project is no longer tested against older PHP versions (which might be okay). I would suggest at least adding the required PHP version. Many would consider this to be a breaking change.

As an alternative, it looks like most of your changes would also be compatible with older PHPUnit versions. Perhaps require something like `^8 || ^5.7` or some other version range that you feel comfortable with? (This would be my preference because it doesn't break compatibility with any existing installations)

Reply:
Thanks for this - did not realize 5.7 would work. Have added that and the travis tests back in.